glommio/src/lib.rs

```diff
@@ -23,6 +18,9 @@
 //! This library provides abstractions for timers, file I/O and networking plus support for
 //! multiple-queues and an internal scheduler, all without using helper threads.
 //!
+//! A more detailed exposition of Glommio's architecture is ]available in this blog
```

Comment:
`]available` should be `[available` instead.

Reply:
tnx, fixed